scripts/animatediff_ui.py

```diff
@@ -10,6 +10,7 @@
 from scripts.animatediff_i2ibatch import animatediff_i2ibatch
 from scripts.animatediff_lcm import AnimateDiffLCM
 
+supportedSaveFormats = ["GIF", "MP4", "WEBP", "WEBM", "PNG", "TXT"]
```

Comment:
I think it makes more sense to put supportedSaveFormats in animatediff_output.py

Reply:
name your variable in snake-case way. after that this PR will be merged to master.